internal/generate/generate.go

```diff
@@ -83,7 +83,7 @@ func WriteMethods(p *packages.Package, ms method.Set, file string, wo ...WriteOp
 		fn(opts)
 	}
 
-	f := jen.NewFile(p.Name)
+	f := jen.NewFilePath(p.PkgPath)
```

Comment:
Could you please explain with a comment the behavior difference between `jen.NewFile` and `jen.NewFilePath`? My understanding is that we need `NewFilePath` so that we can communicate the package to the library (`jennifer`) not just the package name.

Reply:
Done!